CMakeLists.txt

```diff
@@ -29,6 +29,10 @@ if(Kokkos_ENABLE_HIP AND NOT ARBORX_ENABLE_ROCTHRUST)
   message(WARNING "rocThrust is NOT enabled.\nThis will negatively impact performance on AMD GPUs.")
 endif()
 
+if(Kokkos_ENABLE_CUDA OR (Kokkos_ENABLE_HIP AND ARBORX_ENABLE_ROCTHRUST))
+  set(ARBORX_ENABLE_THRUST ON)
+endif()
```

Comment:
How worried should we be that rocThrust stays in sync with upstream Thrust?
Especially with respect to how it is distributed with ROCm and appropriately configured for host backends.

Reply:
We use a limited set of features right now, present in both Thrust and rocThrust. So, at least with respect to that, I think we should not be worried due to the desire of libraries to preserve backwards compatibility, including the host backends.

I see your point, though, and don't have a good answer.